### DISCUSSION TITLE
NewsletterHub

### DISCUSSION THREAD

**Discussion by SimonAKing** [DIYgod/RSSHub#11130] [Ideas]
RSSHub is a great project, can we add the push function of newsletter on the basis of RSSHub? The cost of development should not be too high.

**Comment by sakurayang** [comment]
humm, rsshub is just a RSS feed aggregator. push function of newletter seems like a client function.

I found a [project](https://github.com/WayneGongCN/rss-mailer) which can send RSS Feeds to email.

may be it's a solution for u?
